rllib/execution/train_ops.py

```diff
@@ -184,6 +190,13 @@ def multi_gpu_train_one_step(algorithm, train_batch) -> Dict:
     algorithm._counters[NUM_ENV_STEPS_TRAINED] += train_batch.count
     algorithm._counters[NUM_AGENT_STEPS_TRAINED] += train_batch.agent_steps()
 
+    if algorithm.reward_estimators:
+        learner_info[DEFAULT_POLICY_ID]["off_policy_estimation"] = {}
+        for estimator in algorithm.reward_estimators:
+            learner_info[DEFAULT_POLICY_ID]["off_policy_estimation"][
+                estimator.name
+            ] = estimator.train(train_batch)
```

Comment:
Ideally this would have directly been a callback of some sorts. Is there really no callback we can use here? 

In a separate PR can you add a callback to default callbacks class and then add this as a callback that we inject at runtime

Reply:
There's an `on_learn_on_batch` callback that could've worked, except it takes in a policy (not algorithm) and gets called at the beginning of policy.learn_on_batch. We can add a new callback, or maybe make this part of Postprocessing.

Reply:
Yup, no difference.